packages/esm-admin-openconceptlab-app/src/import/import.component.tsx

```diff
@@ -0,0 +1,176 @@
+import { showNotification, showToast } from '@openmrs/esm-framework';
+import {
+  Button,
+  ButtonSkeleton,
+  Column,
+  FileUploaderDropContainer,
+  FileUploaderItem,
+  FileUploaderSkeleton,
+  Form,
+  Row,
+  SkeletonText,
+} from 'carbon-components-react';
+import React, { useCallback, useEffect, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import styles from './import.component.scss';
+import { startImportWithFile, startImportWithSubscription, useSubscription } from './import.resource';
+
+const Import: React.FC = () => {
+  const { t } = useTranslation();
+  const [isSubscriptionAvailable, setIsSubscriptionAvailable] = useState(false);
+  const [file, setFile] = useState<File>();
+
+  const { data: subscription, isLoading, isError } = useSubscription();
+
+  useEffect(() => {
+    if (!isLoading) {
```

Comment:
What happens when `isError` is true?

Reply:
That sounds great! For error messages, follow [this design](https://app.zeplin.io/project/60d59321e8100b0324762e05/screen/62c6d17f3cc28c113f1bff97).

Reply:
Updated with a409ff1